package/python-kiwi-spec-template

```diff
@@ -19,34 +19,32 @@
 
 # If they aren't provided by a system installed macro, define them
 %{!?_defaultdocdir: %global _defaultdocdir %{_datadir}/doc}
-%{!?__python2: %global __python2 /usr/bin/python2}
 %{!?__python3: %global __python3 /usr/bin/python3}
 
-# Expanded form required for debbuild's simpler engine
-%if %{undefined python2_sitelib}
-%global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
-%endif
-
 %if %{undefined python3_sitelib}
 %global python3_sitelib %(%{__python3} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
 %endif
 
+%if 0%{?el7}
+%global python3_pkgversion 36
+%else
+%{!?python3_pkgversion:%global python3_pkgversion 3}
+%endif
+
 %if 0%{?debian} || 0%{?ubuntu}
 %global is_deb 1
 %global pygroup python
 %global sysgroup admin
 %global develsuffix dev
-%global update_alternatives %{_bindir}/update-alternatives
 %else
 %global pygroup Development/Languages/Python
 %global sysgroup System/Management
 %global develsuffix devel
-%global update_alternatives %{_sbindir}/update-alternatives
 %endif
 
 Name:           python-kiwi
 Version:        %%VERSION
-Provides:       kiwi-schema = 6.6
+Provides:       kiwi-schema = 7.1
```

Comment:
Wow, we were behind the published schema here, right? Unrelated with this PR, but probably we could parse this value from `k.image.schemaversion` in `kiwi/schema/kiwi.rnc`, so we have the schema version in a single place.

Reply:
I realized the provides on accident and fixed it with this PR. You are right it was not related but as we touched the spec I thought this should be corrected. The provides tag here was exclusively used by the yast image builder module. This was a project that is dead since a long time. Actually I have thought to delete this provides but as I don't know who might use it, I just kept it.

The idea to put that setting into some automation is good, but let's do it in a separate PR